clients/spring-boot-starter-camunda-sdk/src/main/resources/modes/saas.yaml

```diff
@@ -2,7 +2,7 @@ camunda:
   client:
     mode: saas
     cloud:
-      base-url: camunda.io
+      domain: camunda.io
```

Comment:
This change needs to be applied to all templated strings (token url, audience, rest address, grpc address)